src/wordpress-recommendations.lib.php

```diff
@@ -26,70 +25,244 @@
+            // phpcs:disable Generic.Files.LineLength
+            $recommendations['forgottenExtensionFestival'] = array(
+                __('Remove unwanted/unused extensions', 'sucuri-scanner') => __('Keeping unwanted themes and plugins increases the chance of a compromise, even if they are disabled.', 'sucuri-scanner'),
+            );
+            // phpcs:enable
+        }
+
+        /*
+         * Check for too much plugins.
+         * @see https://sucuri.net/guides/wordpress-security/#apt
+         */
+        if (count($wpPluginsInstalled) > 50 && !is_multisite()) {
+            // phpcs:disable Generic.Files.LineLength
+            $recommendations['tooMuchPlugins'] = array(
+                __('Decrease the number of plugins', 'sucuri-scanner') => __('The greater the number of plugins installed, the greater the risk of infection and performance issues.', 'sucuri-scanner'),
+            );
+            // phpcs:enable
+        }
+
+        /*
+         * Check for backup plugins.
+         * @see https://sucuri.net/guides/wordpress-security/#bup
+         *
+         * NOTE: This check uses $wpPluginsInstalledName which is created by
+         * the 2FA plugin check.
+         */
+        // phpcs:disable Generic.Files.LineLength
+        $wpBackupsPluginsInstalled = preg_grep('/(Backup|Back-up|Migration|VaultPress|Duplicator|Snapshot|ManageWP|iControlWP|Staging|Updraft|Backwp|Recovery)/i', $wpPluginsInstalledName);
+        if (empty($wpBackupsPluginsInstalled)) {
+            $recommendations['lackOfBackupsPlugins'] = array(
+                __('Install a backup plugin', 'sucuri-scanner') => __('A good set of backups can save your website when absolutely everything else has gone wrong.', 'sucuri-scanner'),
+            );
+        }
+        // phpcs:enable
+
+        /*
+         * Check if File Editing is still possible via wp-admin.
+         * @see https://sucuri.net/guides/wordpress-security/#appconf
+         */
+        if (!defined('DISALLOW_FILE_EDIT')) {
+            // phpcs:disable Generic.Files.LineLength
+            $recommendations['fileEditStillEnabled'] = array(
+                __('Disable file editing', 'sucuri-scanner') => __('Disabling file editing helps preventing an attacker from changing your files through WordPress backend.', 'sucuri-scanner'),
+            );
+            // phpcs:enable
+        }
+
+        /*
+         * Check if WordPress Debug Mode is set.
+         * @see https://wordpress.org/support/article/debugging-in-wordpress/
+         */
+        if (!defined('WP_DEBUG') || WP_DEBUG === true) {
+            // phpcs:disable Generic.Files.LineLength
+            $recommendations['wpDebugOnline'] = array(
+                __('Disable WordPress debug mode', 'sucuri-scanner') => __('The debug mode will cause all PHP errors, notices and warnings to be displayed which can expose sensitive information.', 'sucuri-scanner'),
```

Comment:
If WP_DEBUG is not defined then it defaults to false. In this case we probably don't want to show this recommendation.

Also, I think it would be good to mention `WP_DEBUG` in the message